lib/sync.js

```diff
@@ -95,6 +97,11 @@ function Sync(src, target, opts, callback) {
     };
   }
 
+  if (opts.live) {
+    this.push.on('complete', self.pull.cancel.bind(self.pull));
+    this.pull.on('complete', self.push.cancel.bind(self.push));
+  }
+
```

Comment:
Shoot, missed that. I thought it was in a test instead.

Why does `sync` need to auto-cancel but `replicate` doesn't?